packages/mongodb-memory-server-core/src/MongoMemoryServer.ts

```diff
@@ -201,20 +251,37 @@ export default class MongoMemoryServer {
     return uri;
   }
 
+  /**
+   * Get a mongodb-URI for a different DataBase
+   * @param otherDbName Set this to "true" to generate a random DataBase name, otherwise a string to specify a DataBase name
+   */
   async getConnectionString(otherDbName: string | boolean = false): Promise<string> {
+    // should this function be marked deprecated? because it is just a pass-through to getUri
```

Comment:
should this function be marked `deprecated` because it is just a pass-through to `getUri`?

Reply:
We may deprecate it in the future.
Firstly need to fix README.md to use `getUri()` and after several months deprecate this method.

But really I don't want for now force people to modify their code even via soft deprecate message.

Reply:
changed it in e7eb86d84ea9c19a570f4d2698074904b317f6b4